docs/docs/manual/grelfunctions.md

```diff
@@ -0,0 +1,547 @@
+
+###### get(a, n from, n to (optional))
+Returns a sub-array of a given array, from the first index provided and up to and excluding the optional last index provided. Remember that array objects are indexed starting at 0. 
+
+If to is omitted, only one array item is returned, as a string, instead of a sub-array. To return a sub-array from one index to the end, you can set the to argument to a very high number such as `value.get(2,999)` or you can use something like `with(value,a,a.get(1,a.length()))` to count the length of each array.
+
+Also works with strings; see [get() in String functions](#gets-n-from-n-to-optional).
+
+###### inArray(a, s)
+Returns true if the array contains the desired string, and false otherwise.
+
+###### reverse(a)
+Reverses the array. For example, `[ 0, 1, 2, 3].reverse()` returns the array [ 3, 2, 1, 0 ].
+
+###### sort(a)
+Sorts the array in ascending order. Sorting is case-sensitive, uppercase first and lowercase second. For example, `[ "al", "Joe", "Bob", "jim" ].sort()` returns the array [ "Bob", "Joe", "al", "jim" ]. 
+
+###### sum(a)
+Return the sum of the numbers in the array. For example, `[ 2, 1, 0, 3].sum()` returns 6.
+
+###### join(a, sep)
+Joins the items in the array with sep, and returns it all as a string. For example, `[ "and", "or", "not" ].join("/")` returns the string “and/or/not”.
+
+###### uniques(a)
+Returns the array with duplicates removed. Case-sensitive. For example, `[ "al", "Joe", "Bob", "Joe", "Al", "Bob" ].uniques()` returns the array [ "Joe", "al", "Al", "Bob" ]. 
+
+As of OpenRefine 3.4.1, uniques() reorders the array items it returns; in 3.4 beta 644 and onwards, it preserves the original order (in this case, [ "al", "Joe", "Bob", "Al" ]). 
+
+## Date functions
+If you have a column full of dates in a more common format (such as yyyy/mm/dd or YYYYMMDD) you can convert these using [“Transform” > “Common transforms …” > “to date”](cellediting#data-type-transforms), or the GREL function toDate(). You may wish to create a new column with the transformation, in order to also preserve the more human-readable version.
+
+OpenRefine uses [Date.parse()](https://www.w3schools.com/jsref/jsref_parse.asp) to recognize a variety of formats and convert them, including converting from other time zones to UTC:
+
+![A screenshot of different date formats being converted, and one error.](/img/dates.png)
+
+You may need to do some reformatting if your dates are not being recognized by the toDate() function. For example, in the image above, the date that includes “7AM” is giving an error message, but the ones with “2:42 PM” and “3:22PM” are being converted.
+
+###### now()
+
+Returns the current time in the [ISO 8601 extended format](exploring#data-types). For example, 12:58pm on October 28th 2020 returns [date 2020-10-28T16:58:32Z].
```

Comment:
Should we say this is the current time from the system clock converted to UTC ?

Reply:
There's always 3 things about date functions in OpenRefine... the **datetime**, the **timezone**, the **format**.  I think mentioning that in the #data-type-transforms section would be wise.  And then here in Expressions -> GREL, I agree with Owen to simplify.
In other words, we should have already explained the overarching functionality of Date handling to users already under Transforming data -> Cell editing -> Data-type transforms.  Dates are important enough that they could have their own subsection under #common-transforms  (but in that section, I would still avoid tutorial-ly language, and only explain how dates are treated in OpenRefine, their 3 constituent parts that users have control over, and then provide links to the Technical docs.

Reply:
@allanaaa Also, #data-types and other sections needs a clean up pass for code/syntax highlighting.  I see "null", null, and true, "true", `true`, etc.  But I think that's part of your 2nd contract (extension) work?

Reply:
Yes, I think dates are worth further expansion somewhere. That bit of text has been moved around a bit as I wasn't sure where to drop it. Maybe under overview > data types, rather than under the transforms.